src/EnergyPlus/DaylightingManager.cc

```diff
@@ -6383,8 +6381,8 @@ namespace DaylightingManager {
         static Array2D<Real64> SFSKHR(2, 4); // Sky source luminance factor for sky type (second index),
         //   bare/shaded window (first index)
         static Vector2<Real64> SFSUHR; // Sun source luminance factor for bare/shaded window
-        Array1D<Real64> GLRNDX;        // Glare index at reference point
-        Array1D<Real64> GLRNEW;        // New glare index at reference point
+        static Array1D<Real64> GLRNDX; // Glare index at reference point
+        static Array1D<Real64> GLRNEW; // New glare index at reference point
```

Comment:
Also made these 3 arrays static. I'm sure there could be more work in this function but these changes meet the needs to avoid the crash.

Reply:
Is there any chance that the newly static nature of these arrays could cause problems for other unit tests?  These will stay allocated, etc., between multiple unit tests.

Reply:
Static function members should really be avoided. I'd move them up to the namespace like we have for many other instances like this.

Reply:
Not sure what you guys mean. There were several local arrays that were already static inside this function (a function which is not static, the hh file shows `void DayltgInteriorIllum(int &ZoneNum);`. I just made these local arrays static as well. And the existing static arrays are not deallocated in the clear_state function. Should they be? So what is the recommendation here?

Reply:
Oh, and as far as the daylighting unit test go, none were calling this function but my change to one unit test does now, but it's the only one. So I think I get what you mean now. That means those existing static arrays would have caused a problem but unit testing never saw that since none called this function.